app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -2,15 +2,8 @@
 
 import { TextInput, Button, Box } from '@mantine/core';
 import { useForm } from '@mantine/form';
-import { useEffect, Dispatch, SetStateAction, useMemo } from 'react';
+import { Dispatch, SetStateAction, useMemo } from 'react';
```

Comment:
Are we using Dispatch and SetStateAction in this component?

Reply:
I'm not sure about Dispatch and SetStateAction as it wasn't related to Oauth flow. 
But i can see that Dispatch and SetStateAction are used here
`interface PrelaunchSignUpFormProps {
  setFlowState: Dispatch<SetStateAction<FlowState>>;
}`.

Reply:
I see. Maybe better keep it there for now.